src/components/Input/DropDown/DropDownInput.js

```diff
@@ -70,10 +66,16 @@ class DropDownInput extends React.Component {
       this.props.isFocused &&
       this.SelectedElement.current
```

Comment:
Can we move this validation out of this conditional and rely on the logic within this.focus instead